src/html.ml

```diff
@@ -100,6 +96,24 @@ let text_of_inline ast =
   Text.inline b ast;
   Buffer.contents b
 
+let print_attributes b attributes =
+  begin
+    match attributes.id with
+    | None -> ()
+    | Some s -> Buffer.add_string b (Printf.sprintf " id=\"%s\"" s)
+  end;
+  if List.length attributes.classes <> 0
+  then
+    begin
+      Buffer.add_string b (Printf.sprintf " class=\"%s\"" (String.concat " " attributes.classes))
+    end;
+  if List.length attributes.attributes <> 0
+  then
+    begin
+      let f (d, v) = Buffer.add_string b (Printf.sprintf " data-%s=\"%s\"" d v) in
```

Comment:
Why always prefix attributes with `data-`? This seems a bit ad-hoc.

Reply:
I was using pandoc as a reference for the behaviour of the attributes as there is no standard. But indeed, it could be better without the automatic data prefix